.circleci/soltest.sh

```diff
@@ -36,7 +36,7 @@ set -e
 
 OPTIMIZE=${OPTIMIZE:-"0"}
 EVM=${EVM:-"invalid"}
-REPODIR="$(realpath $(dirname $0)/..)"
+REPODIR="$(realpath "$(dirname "$0")/..")"
```

Comment:
I guess this does work but, I'm slightly confused about double quotes inside double quotes not causing problems

Reply:
Ah, it's probably fine because it's inside a `$()`

Reply:
Yeah, that's perfectly fine in Bash. Might be one of the reasons why `$()` is recommended over ``` `` ```. Not sure if you can nest quotes inside ``` `` ``` but the stuff inside `$()` is basically a separate scope, even if you use it inside quotes.